<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewwindow</class>
 <widget class="QWidget" name="viewwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Read</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>541</width>
     <height>381</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="confirmbutton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>140</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ПРОСМОТР</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>house</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>customer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>booking</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>выберите таблицу:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backbutton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>440</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ВЫХОД</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>фильтр:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Нет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>house_id</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>price</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>house_type</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>city</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>address</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>431</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
